INSTALL.md

```diff
@@ -0,0 +1,11 @@
+# Installation
+
+## Using Docker
+
+Download and install the wrapper script :
+
+```sh
+curl https://raw.githubusercontent.com/constructions-incongrues/masterbizor/master/wrapper > masterbizator-wrapper
```

Comment:
Do you plan on keeping your branch open as the source for the docker file? Or should these and other references be updated to point to my repository?

Reply:
Ooop :)

```suggestion
curl https://raw.githubusercontent.com/fracai/masterbizor/master/wrapper > masterbizator-wrapper
```